sdk/cosmosdb/cosmos/src/auth.ts

```diff
@@ -75,7 +75,7 @@ export async function setAuthorizationTokenHeaderUsingMasterKey(
   resourceType: ResourceType,
   headers: CosmosHeaders,
   masterKey: string
-): Promise<void> {
```

Comment:
Can you please explain what the real gap is and how is it addressed? 
Let's capture it on the PR description.

Reply:
The real gap where? About what?

Reply:
Agree with @kirankumarkolli. @jay-most why there is no PR template filled for this ? It has all the detailed information about package impacted, Change log , design , test cases etc. So that it will be easy for us to navigate which issues are addressed with this PR